app/svelte/src/client/preview/render.ts

```diff
@@ -1,7 +1,8 @@
 import { document } from 'global';
 import dedent from 'ts-dedent';
+import { MountViewArgs, RenderMainArgs } from './types';
 
-let previousComponent = null;
+let previousComponent: any = null;
```

Comment:
I think there's a `Component` type for this?

Reply:
Will look into it :)